Formula/app-engine-java.rb

```diff
@@ -1,14 +1,28 @@
 class AppEngineJava < Formula
   desc "Google App Engine for Java"
-  homepage "https://cloud.google.com/appengine/docs/java/gettingstarted/introduction"
-  url "https://storage.googleapis.com/appengine-sdks/featured/appengine-java-sdk-1.9.32.zip"
-  sha256 "1e757250acb1ecb61a09854bd07ee82a781cdb1fa76c44ed056b4de73a4cff79"
+  homepage "https://cloud.google.com/appengine/docs/java/"
+  url "https://storage.googleapis.com/appengine-sdks/featured/appengine-java-sdk-1.9.42.zip"
+  sha256 "3003892e89fb32a51dc3e2d91658fc7ba8b31f8b0b3fc22ccbd856ed94a03424"
 
   bottle :unneeded
 
+  depends_on :java => "1.7+"
+
   def install
     rm Dir["bin/*.cmd"]
     libexec.install Dir["*"]
-    bin.write_exec_script Dir["#{libexec}/bin/*"]
+    bin.write_exec_script %w[appcfg.sh dev_appserver.sh endpoints.sh google_sql.sh].map { |fn| libexec/"bin/#{fn}" }
```

Comment:
Can you use `.each` instead of `.map` here just so it's more readable for non-Rubyists, thanks.